app/models/host.rb

```diff
@@ -56,8 +56,8 @@ def test_results?(_filter, _operator, value)
   end
 
   def update_from_inventory_host!(i_host)
-    update!(name: i_host['display_name'],
-            os_major_version: i_host['os_major_version'],
-            os_minor_version: i_host['os_minor_version'])
+    update!({ name: i_host['display_name'],
+              os_major_version: i_host['os_major_version'],
+              os_minor_version: i_host['os_minor_version'] }.compact)
```

Comment:
Ok, this change is straightforward. Just remove `nil` values completely.

Reply:
So the default value in db for them will be 0?

Reply:
No, the default value in the DB is `nil`.